examples/deployment/kubernetes/README.md

```diff
@@ -22,28 +26,26 @@ deployment on Google Cloud using Kubernetes and Cloud Spanner.
 ## Process
 
 1. Go to [Google Cloud Console](https://console.cloud.google.com)
-1. Create a new project
-1. Edit the [example-config.sh](example-config.sh) file, set `PROJECT_ID` to
-   the ID of your project
-1. Run: `./create.sh example-config.sh`.
-   This script will create the Kubernetes cluster, node pools, and Spanner
-   database, service account and etcd cluster.
-   It should take about 5 to 10 minutes to finish and must complete without
-   error.
-1. Now you can deploy the Trillian services.
-   Run: `./deploy.sh example-config.sh`
-   This will build the Trillian Docker images, tag them, and create/update the
-   Kubernetes deployment.
-1. To update a running deployment, simply re-run `./deploy.sh example-config.sh`
-   at any time.
+1. Create a new project and copy its [Project
+   ID](https://cloud.google.com/resource-manager/docs/creating-managing-projects#identifying_projects).
+1. Create the infrastructure: `terraform init && terraform apply -var="gcp_project=PROJECT_ID"`.
+   It is ok to re-execute this command at any time.
+1. The script will output a gcloud command similar to:
+   ```
+   gcloud config set project PROJECT_ID && \
+   gcloud container clusters get-credentials cluster --region=us-west1 && \
+   gcloud builds submit --config=cloudbuild.yaml ../../..
+   ```
+   Execute this set of commands to get the credentials for your new cluster and build
+   the docker images.
```

Comment:
I wonder whether we'd be best off not building the Docker images but instead using the pre-built ones at http://gcr.io/trillian-opensource-ci/? That'd simplify things, with the trade-off of being less useful for devs working on Trillian (but they can use docker-compose).

Reply:
Yes, I agree, this makes the deployment easier. To which images should we point to? The latest release (1.3.3) or latest build (latest)? 
We can keep docker || gcloud build as an optional step?

Reply:
The latest release would be best, since we have greater confidence in those than the latest build.